proxy-wasm-jaxrs-quarkus-example/src/test/java/io/roastedroot/proxywasm/jaxrs/example/DispatchCallOnTickTest.java

```diff
@@ -0,0 +1,40 @@
+package io.roastedroot.proxywasm.jaxrs.example;
+
+import static io.roastedroot.proxywasm.jaxrs.example.Helpers.assertLogsContain;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.roastedroot.proxywasm.StartException;
+import io.roastedroot.proxywasm.jaxrs.WasmPlugin;
+import io.roastedroot.proxywasm.jaxrs.WasmPluginFeature;
+import jakarta.inject.Inject;
+import org.junit.jupiter.api.Test;
+
+@QuarkusTest
+public class DispatchCallOnTickTest {
+
+    @Inject WasmPluginFeature feature;
+
+    @Test
+    public void test() throws InterruptedException, StartException {
+        WasmPlugin plugin = feature.pool("dispatchCallOnTickTest").borrow();
+        assertNotNull(plugin);
+
+        var logger = (MockLogger) plugin.logger();
+        Thread.sleep(300);
```

Comment:
Would it be possible to trigger `plugin.close()` instead waiting here?

Reply:
We basically have to wait for the tick events to get delivered to the plugin. . They are getting delivered every 100 ms (configured by the plugin).   If we want avoid the wait, we would need to implement a new wasm module, avoiding the use of the async tick event to trigger the http call.

Reply:
I see, thanks for explaining, (the config is not in the diff).
A `Thread.sleep` always makes me a little anxious should we use Awaitility or something similar to avoid this becoming flaky over time?

Reply:
Yeah.. I think I might just implement a better wasm plugin that allows us to test things better.